### PR TITLE
WWL initial scaffolding & architecture

### PR DESCRIPTION
**Pull request by gvn**
Related to #3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,37 +1,3 @@
...
-# Dependency directories
-node_modules
-jspm_packages
-
-# Optional npm cache directory
-.npm
-
-# Optional REPL history
-.node_repl_history
+.DS_Store
```
Comment (gideonthomas) [inline .gitignore:1]: can we make this `**/.DS_Store`? I've had times when it pops up in sub-directories too :(

File: package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "womenandweblit.mozilla.org",
```
Comment (gideonthomas) [inline package.json]: This will change to `womenandweb`

File: package.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  "version": "1.0.0",
+  "description": "Women and Web Literacy",
+  "scripts": {
+    "autoprefix": "node scripts/autoprefix.js",
+    "build:folders": "shx mkdir -p dest/css",
```
Comment (gideonthomas) [inline package.json:8]: Suggestion: Can we add line breaks between related tasks? I remember seeing it in another project and it looked purrrty. (Feel free to ignore though)

File: .gitignore (hunk 1 above)
Comment (gvn) [inline .gitignore:1]: This should work for all `.DS_Store` files no matter what directory.

File: scripts/autoprefix.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+let autoprefixer = require(`autoprefixer`);
```
Comment (gideonthomas) [inline scripts/autoprefix.js:3]: For this to run on node LTS shouldn't this have `use strict;`?

File: source/index.pug [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  head
+    meta(charset='utf-8')
+
+    meta(name='viewport', content='width=device-width, initial-scale=1')
+    meta(name='description', content='Women and Web Literacy')
+
+    meta(property='og:type', content='website')
+    meta(property='og:site_name', content='Women and Web Literacy')
+    meta(property='og:locale', content='en_US')
+    meta(property='og:url', content='https://womenandweblit.mozilla.org')
```
Comment (gideonthomas) [inline source/index.pug]: This will need to change to the Github pages url viz. `https://mozilla.github.io/womenandweb`

**Comment by gideonthomas** [comment]
Will the setup/workflow instructions (updated in the README.md) come in a separate PR?

File: scripts/autoprefix.js (hunk 4 above)
Comment (gvn) [inline scripts/autoprefix.js:3]: Not that I know of?

**Comment by gvn** [comment]
Yeah I wanted to do docs as another PR...

**Comment by gvn** [comment]
Ok, did those fixups...

File: scripts/autoprefix.js (hunk 4 above)
Comment (gideonthomas) [inline scripts/autoprefix.js:3]: Running this on node 4.4 fails with:

```
SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
```

**Comment by gideonthomas** [comment]
@gvn just that one comment about strict mode, r=me with that

**Comment by gvn** [comment]
Strict mode shouldn't be necessary.

**Comment by gideonthomas** [comment]
Does it work for you on Node 4.4 without strict mode? 
(On the other hand, since we're not running a node production server, maybe we don't care about supporting node lts 👍 )

**Comment by gideonthomas** [comment]
r+

**Pull request merged**